src/management/apps.ts

```diff
@@ -224,3 +234,66 @@ function getListAppsResourceString(projectId: string, platform: AppPlatform): st
 
   return `/v1beta1/projects/${projectId}${resourceSuffix}`;
 }
+
+/**
+ * Gets the configuration artifact associated with the specified a Firebase app
+ * @return a promise that resolves to configuration file content of the requested app
+ */
+export async function getAppConfig(
+  appId: string,
+  platform: AppPlatform
+): Promise<AppConfigurationData> {
+  try {
+    const response = await api.request("GET", getAppConfigResourceString(appId, platform), {
+      auth: true,
+      origin: api.firebaseApiOrigin,
+      timeout: TIMEOUT_MILLIS,
+    });
+    return parseConfigFromResponse(response.body, platform);
+  } catch (err) {
+    logger.debug(err.message);
+    throw new FirebaseError(
+      `Failed to get ${platform} app configuration. See firebase-debug.log for more info.`,
+      {
+        exit: 2,
+        original: err,
+      }
+    );
+  }
+}
+
+function getAppConfigResourceString(appId: string, platform: AppPlatform): string {
+  let platformResource;
+  switch (platform) {
+    case AppPlatform.IOS:
+      platformResource = "iosApps";
+      break;
+    case AppPlatform.ANDROID:
+      platformResource = "androidApps";
+      break;
+    case AppPlatform.WEB:
+      platformResource = "webApps";
+      break;
+    default:
+      // This should never happen
```

Comment:
Of course it shouldn't. So remove the comment, please. (same below)

Reply:
Done